### DISCUSSION TITLE
卡在 Upgrading database... from v1.23.2 to v1.37.58 please wait

### DISCUSSION THREAD

**Discussion by rayosu** [electerm/electerm#3168] [Q&A] [answered]
![image](https://github.com/electerm/electerm/assets/3970647/fe451ab4-d0ae-43bd-9558-d0d6f44fb37f)
MacOS x64版本，卸载重装还是提示这个

![image](https://github.com/electerm/electerm/assets/3970647/156fcd39-5b3f-434d-8f9e-b41c1b5d077d)
而旧版本又连不上ssh

**Comment by rayosu** [comment]
updateSyncSetting = function(i) {
        const {store: e} = window
          , t = Object.assign({}, ae(e.config.syncSetting), i);
        i.giteeGistId && !i.giteeSyncPassword && delete t.giteeSyncPassword,
        i.githubGistId && !i.githubSyncPassword && delete t.githubSyncPassword,
        e.setConfig({
            syncSetting: t
        })
    }
报错行数应该是这个：e.config.syncSetting，config为null

**Comment by zxdong262** [comment]
可以分享一下```~/Library/Application\ Support/electerm/users/default_user/electerm.data.nedb```吗，注意脱敏，然后可以删除这个文件启动试试可以不？

**Comment by zxdong262** [comment, marked as answer]
我知道原因了，不用分享了，感谢反馈，将在下个版本修复，不过你还是需要删除~/Library/Application\ Support/electerm/users/default_user/electerm.data.nedb重启electerm1.37.58应该就可以了
